src/rustdoc-json-types/lib.rs

```diff
@@ -481,12 +481,7 @@ pub enum Term {
 #[serde(tag = "kind", content = "inner")]
 pub enum Type {
     /// Structs, enums, and traits
-    ResolvedPath {
-        name: String,
-        id: Id,
-        args: Option<Box<GenericArgs>>,
-        param_names: Vec<GenericBound>,
```

Comment:
IIUC, you removed this field because it was always empty?

Reply:
Yeah, is was [origionaly ment](https://rust-lang.github.io/rfcs/2963-rustdoc-json.html#kind--resolved_path) to be part of handling `dyn` traits, but was never used, and we handle dyn differently now (#99787)